sdk/monitor/azure-monitor-query/tests/test_metrics_client.py

```diff
@@ -15,9 +16,13 @@ def _credential():
 def test_metrics_auth():
     credential = _credential()
     client = MetricsQueryClient(credential)
-    # returns LogsQueryResults 
-    response = client.query(os.environ['METRICS_RESOURCE_URI'], metric_names=["PublishSuccessCount"], timespan='P2D')
-
+    response = client.query(
+        os.environ['METRICS_RESOURCE_URI'],
+        metric_names=["MatchedEventCount"],
```

Comment:
Is it by design to change it from "PublishSuccessCount" to "MatchedEventCount"?

Reply:
the metric name publishe success count does not have any aggregations supported - these metric names are defined by each service